programFiles/may.cpp

```diff
@@ -0,0 +1,327 @@
+#include<iostream>
+#include<vector>
+#include<fstream>
+#include<string>
+#include<sstream>
+#include<cmath>
+double cRound(double number){

```

Comment:
An error may occur in the cRound method if the number has a fractional part greater than or equal to 10. In this case, the function will return an incorrect result. It is better to use the standard round function from the <cmath> library.

Reply:
Custom round function, instead of the round function from "cmath" library was used to prevent getting "-0", because library function doesn't clear "sign bit".